.github/workflows/PR.yml

```diff
@@ -24,8 +24,10 @@ jobs:
   build:
     strategy:
       # Ensure any changes to the matrix are also made in PR-skip.yml
+      # macOS-latest (Intel) is required to run Android simulator tests (instrumentation)
+      # due to nested virtualization not supported on M1
       matrix:
-        os: [ macOS-latest, windows-latest, ubuntu-latest ]
+        os: [ macOS-14, macOS-latest, windows-latest, ubuntu-latest ]
```

Comment:
We should move the emulator tests to an ubuntu runner, like in Redwood: https://github.com/cashapp/redwood/blob/trunk/.github/workflows/build.yaml#L29-L49

Reply:
Thanks for the hint, I tested it: https://github.com/hfhbd/kotlinx-uuid/pull/311 and its also faster (due the 4 core and more ram)